hsa-rocr/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# Maintainer: acxz <akashpatel2008 at yahoo dot com>
+# Contributor: Jakub Okoński <jakub@okonski.org>
+# Contributor: Ranieri Althoff <ranisalt+aur at gmail.com>
+
+pkgname=hsa-rocr
+pkgver=3.1.0
+pkgrel=1
+pkgdesc='ROCm Platform Runtime: ROCr a HPC market enhanced HSA based runtime'
+arch=('x86_64')
+url='https://github.com/RadeonOpenCompute/ROCR-Runtime'
+license=('custom:NCSAOSL')
+makedepends=('cmake' 'libelf' "hsakmt-roct>=$pkgver")
+provides=("rocr-runtime=$pkgver")
+replaces=('rocr-runtime')
+conflicts=('rocr-runtime')
+source=("$url/archive/roc-$pkgver.tar.gz")
+sha256sums=('b162464ef87ce39518e59ef8406d6b897aa7a930795c586829614ed87aa1c2ce')
+_dirname="$(basename $url)-roc-$pkgver"
```

Comment:
This is nice because we can copy and paste to every package and it will work out of the box usually :)

Reply:
True